console/console-init/ui/src/modules/messaging-project/components/EditMsgProject/EditMsgProject.tsx

```diff
@@ -45,22 +45,22 @@ export const EditMsgProject: React.FC<IEditMsgProjectProps> = ({
   return (
     <Modal
       variant="large"
-      id="edit-msg-edit-modal"
+      id="edit-msg-project-modal"
```

Comment:
For consistency, either write messaging-project or msg-project.

Reply:
I have started ID's with the name of the file, which in this case is edit-msg-project, thus the ID. That is the consistent format I have followed